package.json

```
@@ -19,17 +19,12 @@
     "web-vitals": "^1.0.1"
   },
   "scripts": {
+    "lint": "npx eslint ./src",
```

nit: I think npx isn't required as eslint is an implicit dependency for react-scripts: https://github.com/facebook/create-react-app/blob/main/packages/react-scripts/package.json#L47

Also suggest adding a fix shortcut:
```suggestion
    "lint": "eslint ./src",
    "lint:fix": "eslint ./src --fix",
```

---

No. npx is required if eslint is not installed globally. I use "npx eslint ..." since it suits for both the two cases - local installation and global installation of eslint.

I added the task "lint:fix" per your suggest.

---

Ah, sorry. My previous reply was obsolete. "npx" is a convention to ensure flexible ways to run the script via different integration tools when eslint is installed locally, but npm/yarn can surely locate eslint when they run scripts explicitly. We can follow the simple way as you suggested for now.

I updated package.json to simplify the dependencies of eslint, and removed "npx".